docs/examples/camera_display.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    disp.cmap = "rainbow"
</pre><p>Let's not show how to chose a bad colormap</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/ctapipe/pull/2266/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>